LookingMenu/Scenes/Add Diet Screen/AddDietViewController.swift

```diff
@@ -0,0 +1,232 @@
+import UIKit
+
+private enum Activities: String, CaseIterable {
+    static var allCases: [Activities] {
+        return [ .sedentary,
+                 .light,
+                 .moderate,
+                 .activez,
+                 .very,
+                 .extra]
+    }
+    
+    case sedentary =  "Sedentary: little or no exercise"
+    case light = "Light: exercise 1-3 times/week"
+    case moderate = "Moderate: exercise 4-5 times/week"
+    case activez = "Active: daily exercise or intense exercise 3-4 times/week"
+    case very = "Very Active: intense exercise 6-7 times/week"
+    case extra = "Extra Active: very intense exercise daily, or physical job"
+    
+    var activityValue: Double {
+        switch self {
+        case .sedentary: return 1.232
+        case .light: return 1.411
+        case .moderate: return 1.504
+        case .activez: return 1.591
+        case .very: return 1.771
+        case .extra: return 1.950
+        }
+    }
+}
+
+
+private enum ConstantAddDietView {
+    static let radiusView: CGFloat = 10
+    static let calorieMale = 5
+    static let calorieFemale = -161
+    static let calorieHeight = 6.25
+    static let calorieWeight = 10.0
+    static let calorieAge = 5
+    static let heightDropDownTableCell: CGFloat = 50
+    static let iconCheck = "checkmark.circle.fill"
+    static let iconUnCheck = "xmark.circle.fill"
+    static let idDropDownActivityCell = "DropDownActivityCell"
+    static let textFieldHeightPlaceholder = "Insert in cm"
+    static let textFieldWeightPlaceholder = "Insert in kg"
+    static let textFieldAgePlaceholder = "Insert in year"
+}
+
+final class AddDietViewController: UIViewController {
+    @IBOutlet private weak var heightOfUserTextField: UITextField!
+    @IBOutlet private weak var weightOfUserTextField: UITextField!
+    @IBOutlet private weak var ageOfUserTextField: UITextField!
+    @IBOutlet private weak var checkGenreMaleButton: UIButton!
+    @IBOutlet private weak var checkGenreFemaleButton: UIButton!
+    @IBOutlet private weak var dropDownActivityButton: UIButton!
+    @IBOutlet private weak var dropDownActivityTableView: UITableView!
+    @IBOutlet private weak var createDietButton: UIButton!
+    private let sqlite3 = SQLiteService()
+    private let activities = Activities.allCases
+    private var selectActivity = Activities.sedentary.activityValue
+    private var genreSelected = true
```

Comment:
```suggestion
    private var selectedGender = true
```
genre là thể loại mà nhỉ :D